packages/@ourworldindata/grapher/src/stackedCharts/StackedDiscreteBarChart.tsx

```diff
@@ -364,12 +390,18 @@ export class StackedDiscreteBarChart
         }))
     }
 
+    // useful if `this.barHeight` can't be used due to a cyclic dependency
+    // keep in mind though that this is not exactly the same as `this.barHeight`
+    @computed private get approximateBarHeight(): number {
+        return (0.8 * this.boundsWithoutLegend.height) / this.barCount
+    }
```

Comment:
There is a circular dependency here that I can't get rid of easily. If I remember correctly, it's something like: `labelFontSize` -> `barHeight` -> `innerBounds` -> `labelWidth` -> `labelStyle` -> `labelFontSize`.

Breaking the cycle by using `boundsWithoutLegend` instead of `innerBounds` is not new, but I added a computed property that makes it explicit.